project/app/utils/moderation/text_moderation.py

```diff
@@ -0,0 +1,71 @@
+import io
+from os import path
+
+
+class TextModerationBase:
+    """Base Class for text moderation classes"""
+    def __init__(self, wordlist_file: str):
+        """Arguments:
+        `wordlist_file` str - path to file containing words to be
+        used in moderation"""
+        # wordlist = open file descriptor
+        self._wordlist = wordlist_file
+        # words = a set of words from self._wordlist file descriptor
+        self._words = self.open_words(self._wordlist)
```

Comment:
I don't think the `wordlist =` comment is necessary, since you've just described the `wordlist_file` variable in the docstring. But could you change the second comment to remove the `=` so it doesn't look like a commented line of code? That's what I thought it was at first glance.